src/DependencyInjection/Configuration.php

```diff
@@ -590,6 +590,15 @@ private function addSharedPluginNodes(ArrayNodeDefinition $pluginNode, $disableA
             ->end()
         ->end();
         // End stopwatch plugin
+
+        $error = $children->arrayNode('error');
+        $disableAll ? $error->canBeEnabled() : $error->canBeDisabled();
```

Comment:
to avoid behaviour BC breaks, this should always `canBeEnabled`. otherwise people will suddenly unexpectedly get exceptions